README.md

```diff
@@ -2,3 +2,14 @@
 
 # sermon-helper-py
 Helper files for managing sermon files
+
+
+
+# Usage: 
+(BEFORE USING, MAKE SURE TO INSTALL EYED3 PACKAGE IN PYTHON WITH PIP)- Ask Mark how to do this.
+
+Copy Audacity edited mp3 file to same folder as tagmp3.py.
```

Comment:
Could you add that the file's name has to match the `source` entry in `sermon.ini`?